test/e2e/README.md

```diff
@@ -0,0 +1,61 @@
+# End to End testing
+
+# Upgrade - bump
+
+The current end to end testing pipeline relies on
+* pupernetes
+* argo
+* kinvolk flatcar
+
+## Bump hyperkube version
```

Comment:
It would be nice to add some info about where to update flags passed to the hyperkube binary since an upgrade of hyperkube might require some changes there.